query-algebrizer/Cargo.toml

```diff
@@ -6,6 +6,7 @@ workspace = ".."
 [dependencies]
 enum-set = { git = "https://github.com/rnewman/enum-set" }
 error-chain = "0.8.1"
+maplit = "0.1"
```

Comment:
I think it's macro-only, and it's only 198 LOC. I'm not sure if a macro-only crate can be marked as dev-only… let's find out!